runtime/cudaq/algorithms/observe.h

```diff
@@ -60,9 +60,14 @@ concept ObserveCallValid =
 /// \p shots is the number of shots to run for the given kernel. The default of
 /// -1 means direct calculations for simulation backends. \p noise is the noise
 /// model to use for the observe operation.
+/// \p num_trajectories is the optional number of trajectories to be used when
+/// computing the expectation values in the presence of noise. This parameter is
+/// only applied to simulation backends that support trajectory noisy
+/// simulation.
```

Comment:
```suggestion
/// only applied to simulation backends that support noisy
/// simulation of trajectories.
```
This sounds better to my ear, but I admit the original may be expected in the domain.